packages/edit-site/src/components/page-templates/dataviews-templates.js

```diff
@@ -135,6 +135,15 @@ export default function DataviewsTemplates() {
 						: titleB.localeCompare( titleA );
 				} );
 			}
+			if ( view.sort.field === 'author' ) {
```

Comment:
I had a [TODO right above](https://github.com/WordPress/gutenberg/pull/56333/files#diff-b45cf4395b9ac63b295cc17bdf6b8541d7db52c2745bbd1674254efe7e5188a1R127) because I was expecting this. 

>TODO: Explore how this can be more dynamic..

These two cases are sorting by a `string` a different item prop. I'm wondering if we could generalize this a bit.

Reply:
Hi @ntsekouras, I updated this to be a generic string sorter.